lib/provider/ui/windows/control-panel.tsx

```diff
@@ -0,0 +1,69 @@
+import type { ReactNode } from "react";
+import type { SelectOption } from "react95/dist/Select/Select.types";
+import type { Theme } from "react95/dist/types";
+
+import { useAtom } from "jotai";
+import { useEffect, useState } from "react";
+import { GroupBox, Select } from "react95";
+import styled from "styled-components";
+
+import { themeAtom } from "~/lib/atom";
+import { Window } from "~/lib/window";
+
+type ThemeType = "original" | "blackAndWhite" | "modernDark";
+
+const themeOptions: SelectOption<ThemeType>[] = [
+    { label: "Default", value: "original" },
+    { label: "Black and white", value: "blackAndWhite" },
+    { label: "Modern Dark", value: "modernDark" }
+];
```

Comment:
We should hard-code these values. These values should be dynamically loaded form `react95/dist/themes`.